pkg/systemd/generate/containers.go

```diff
@@ -170,6 +168,7 @@ func executeContainerTemplate(info *containerInfo, options entities.GenerateSyst
 	info.EnvVariable = EnvVariable
 	info.ExecStart = "{{.Executable}} start {{.ContainerNameOrID}}"
 	info.ExecStop = "{{.Executable}} stop {{if (ge .StopTimeout 0)}}-t {{.StopTimeout}}{{end}} {{.ContainerNameOrID}}"
+	info.ExecStopPost = "{{.Executable}} stop {{if (ge .StopTimeout 0)}}-t {{.StopTimeout}}{{end}} {{.ContainerNameOrID}}"
```

Comment:
should it be '-{{.Executable}} stop {{if (ge .StopTimeout 0)}}-t {{.StopTimeout}}{{end}} {{.ContainerNameOrID}}`  (extra - at the beginning) to ignore failures?

Reply:
I think we're good as `stop` doesn't error if a container is already stopped.

Reply:
is there any risk that the container is created with `--rm` so the first stop will also delete it?